src/context/QuizContext.js

```diff
@@ -0,0 +1,42 @@
+import { createContext, useContext, useState } from "react";
+import axios from "axios";
+
+const QuizContext = createContext();
+
+const QuizProvider = ({ children }) => {
+  const [quizzes, setQuizzes] = useState({
```

Comment:
Suggestion: You can use useReducer here, to ease the state management process.

Reply:
Since I am not updating any state value except for the answers so I thought reducer is not required here.